test/integration/site_pr_builder/serverspec/site_pr_builder.rb

```diff
@@ -0,0 +1,13 @@
+require 'serverspec'
+require 'spec_helper'
+
+set :backend, :exec
+
+describe 'site_pr_builder' do
+  it_behaves_like 'jenkins_server'
+end
+
+describe command('curl -k https://127.0.0.1/job/site_pr_builder/ -o /dev/null -v 2>&1') do
+  its(:stdout) { should match(/X-Jenkins-Session:/) }
+  its(:exit_status) { should eq 0 }
+end
```

Comment:
You should check that each job you want to exist actually exists when you curl the site.